temporal/api/cloud/cloudservice/v1/request_response.proto

```diff
@@ -357,8 +358,15 @@ message GetUserGroupsRequest {
     string page_token = 2;
     // Filter groups by the namespace they have access to - optional.
     string namespace = 3;
-    // Filter groups by their name - optional.
-    string group_name = 4;
+    // Filter groups by the display name - optional.
+    string display_name = 4;
+
+    message GoogleGroupFilter {
```

Comment:
Personally I think it's best to put all nested messages at the bottom of the parent, though some proto authors put them at the top, but I don't think it's very common to intersperse between fields.